packages/frontend/src/components/screens/AccountSetupScreen.tsx

```diff
@@ -78,23 +80,23 @@ export default function AccountSetupScreen({
     }
   }, [onKeyDown])
 
+  const onCancel = () => selectAccount(accountId)
+
   return (
     <ImageBackdrop variant='welcome'>
-      <Dialog onClose={() => {}} fixed>
+      <Dialog canOutsideClickClose={false} onClose={onCancel} fixed>
```

Comment:
Isn't `onClose` also invoked when the dialog is closed with a success? I believe originally `onClose={() => {}}` meant to prevent the dialog from being closed when clicking outside. Why was `onClose={onCancel}` added?